sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -1153,6 +1153,12 @@ object SQLConf {
     .booleanConf
     .createWithDefault(true)
 
+  val JSON_GENERATOR_IGNORE_NULL_FIELDS =
+    buildConf("spark.sql.jsonGenerator.nullFields.ignore")
```

Comment:
config name has namespaces, we'd better avoid creating unnecessary namespaces. How about `spark.sql.jsonGenerator.ignoreNullFields`

Reply:
okey, I will change it.